style.css

```diff
@@ -0,0 +1,101 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap');
+*{
+    font-family: 'Montserrat', sans-serif;
+}
+body{
+    background-color: #000000;
+}
+.card-area{    
+    background: linear-gradient(98.76deg, #1C0E0E 24.39%, #1C1833 81.33%);
+    border-radius: 50px;
+}
```

Comment:
You can add a `.search-icon` class and `border-left: 2px solid white` to add the left line to it

Reply:
Added the requested changes. 
Removed
nav div {
    border-left: solid #ffffff;
}